csharp/test/Microsoft.ML.OnnxRuntime.Tests/Tensors/TensorTests.cs

```diff
@@ -160,7 +160,6 @@ public void ConstructFromDimensions(TensorConstructor tensorConstructor)
             Assert.Equal(tensorConstructor.IsReversedStride, tensor.IsReversedStride);
 
             //Assert.Throws<ArgumentNullException>("dimensions", () => tensorConstructor.CreateFromDimensions<int>(dimensions: null));
```

Comment:
> //Assert.Th [](start = 11, length = 12)

Remove commented code  if not needed

Reply:
Actually kept the test - looks like this was checked-in by mistake. The code guarding against null dimensions and this test should both exist.